src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/IncrementalHash.net46.cs

```diff
@@ -116,7 +114,7 @@ public byte[] GetHashAndReset()
                 _hash.Initialize();
             }
 
-            _hash.TransformFinalBlock(s_empty, 0, 0);
+            _hash.TransformFinalBlock(Array.Empty<byte>(), 0, 0);
```

Comment:
I feel like I did this because of an API scan in the net46 compile.  For NetFx we are/were limited to referencing mscorlib:net45 for patching reasons, and the TFS net46 build may be/have been aware of that.

If we think this isn't going to be a problem, it's a good change.  But I don't know how to know a priori if the build failure is still a potential issue.